frontend/app/routes/$lang+/_protected+/personal-information+/index.tsx

```diff
@@ -62,6 +64,7 @@ export default function PersonalInformationIndex() {
   const { personalInformation, preferredLanguage, countryList, regionList, updatedInfo } = useLoaderData<typeof loader>();
   const { i18n, t } = useTranslation(handle.i18nNamespaces);
   const params = useParams();
+  const userOrigin = useUserOrigin();
 
   return (
     <>
```

Comment:
Thoughts on wrapping this page with `<div className="max-w-prose">`?

Reply:
I think it makes it match the figma better. Implemented in commit d1533cc9c6bf3ad4b9c816a7282ee7db500669b8.